src/java/us/kbase/templates/python_server.vm.properties

```diff
@@ -146,12 +147,6 @@ class JSONRPCServiceCustom(JSONRPCService):
         debugging purposes.
         """
         rdata = jsondata
-        # we already deserialize the json string earlier in the server code, no
-        # need to do it again
-#        try:
-#            rdata = json.loads(jsondata)
-#        except ValueError:
-#            raise ParseError
```

Comment:
👍

Reply:
I thought you'd like the removal of commented-out code :)